build_tools/cmake/iree_native_test.cmake

```diff
@@ -89,6 +89,25 @@ function(iree_native_test)
     list(APPEND _RULE_DATA "${_RULE_TEST_INPUT_FILE_ARG}")
```

Comment:
No need to do it right now, but we can probably unify _RULE_TEST_INPUT_FILE_ARG and _RULE_ARGS, so all of them use the new `{{}}` syntax.